core/src/main/java/org/elasticsearch/index/query/QueryParsingException.java

```diff
@@ -44,7 +44,8 @@ public QueryParsingException(QueryParseContext parseContext, String msg, Object.
     }
 
     public QueryParsingException(QueryParseContext parseContext, String msg, Throwable cause, Object... args) {
-        super(parseContext.index(), msg, cause, args);
+        super(msg, cause, args);
```

Comment:
I tried it out and method resolution seems to work fine: https://github.com/bleskes/elasticsearch/commit/6f03cfc4ebeabc1d7cf39bbaaac578f60d6c558f

Just a suggestion though - don't mind too much.